frontend/src/features/Reportings/components/ReportingForm/FormComponents/ThemeSelector/index.tsx

```diff
@@ -29,6 +30,22 @@ export function ThemeSelector({ isLight = true, label, name }) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [year])
 
+  const showVesselMessage = useMemo(() => {
```

Comment:
```suggestion
  const isVesselInformationRequested = useMemo(() => {
```
booléen en `isXXX` ?